### DISCUSSION TITLE
How to get Badges in Github ??

### DISCUSSION THREAD

**Discussion by SiddharthhD** [HarshitKumar9030/Tailwind-Components#10] [Q&A] [answered]
### _**How to get Badges in Github ??**_

They look cool

**Comment by HarshitKumar9030** [comment]
Ok, Try opening an issue by clicking on that issue tab above and close it within 5 min and see the magic on your profile

**Comment by SiddharthhD** [comment]
let me try wait

**Comment by SiddharthhD** [comment]
Woww, I got a badge thank you so much

**Comment by HarshitKumar9030** [comment, marked as answer]
you can see more here https://github.com/Schweinepriester/github-profile-achievements

**Comment by SiddharthhD** [comment]
Thanks
